workflows/yelp.com/workflows.arazzo.json

```diff
@@ -14,9 +14,9 @@
   ],
   "workflows": [
     {
-      "workflowId": "find-business-by-phone-and-get-details",
-      "summary": "Find a business by phone number and then retrieve detailed information about that business.",
-      "description": "Find a business by phone number and then retrieve detailed information about that business.",
+      "workflowId": "find-business-details-by-phone",
+      "summary": "Search for a business by phone number and then retrieve detailed information about it.",
+      "description": "Search for a business by phone number and then retrieve detailed information about it.",
```

Comment:
We need to remove `id` from inputs - doesn't make sense here as we're search a business by phone